sqrl-cli/entrypoint.sh

```diff
@@ -16,8 +16,20 @@
 #
 
 set -e
+set -o pipefail
+
+# Enable debug mode if DEBUG environment variable is set
+if [ "$DEBUG" = "1" ]; then
```

Comment:
Single bracket conditions are an anti-pattern IMO, so we should avoid using it. 

Also, I think it does not matter what value `DEBUG` has, so if it is defined and not empty, we should `set -x`. For that, we could use a condition like this:
```bash
if [[ -n "${DEBUG+x}" && -n "$DEBUG" ]]; then
...
```